server.js

```diff
@@ -15,6 +15,7 @@ app.prepare().then(() => {
     handle(req, res, parsedUrl);
   }).listen(3000, (err) => {
     if (err) throw err;
+    /* trunk-ignore(eslint/no-console) */
```

Comment:
Why is this line needed!?

Reply:
Because ESLint has a no-console rule to avoid people leaving in debug statements, and the line below has a debug statement that we want to leave in (have checked this with Patrick).